Testing doesn't work

```
$ PYTHONPATH=. python3 -mpytest -sxv tests/

Test session starts (platform: linux, Python 3.9.9, pytest 6.0.2, pytest-sugar 0.9.4)
cachedir: .pytest_cache
rootdir: /src/openapi3
plugins: asyncio-0.14.0, xdist-2.2.0, gc-0.0.1, sugar-0.9.4, cov-2.10.1, anyio-3.3.0.1, forked-1.3.0, mock-1.10.4
collecting ... 

――――――――――――――――――――――――――――――――――― ERROR at setup of test_createPet ――――――――――――――――――――――――――――――――――――
ScopeMismatch: You tried to access the 'function' scoped fixture 'unused_tcp_port_factory' with a 'session' scoped request object, involved factories
tests/fastapi_test.py:30:  def server(event_loop, config)
tests/fastapi_test.py:17:  def config(unused_tcp_port_factory)
../../usr/lib/python3/dist-packages/pytest_asyncio/plugin.py:225:  def unused_tcp_port_factory()
                                                                                            4% ▍         
======================================== short test summary info ========================================
FAILED tests/fastapi_test.py::test_createPet
!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!! stopping after 1 failures !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!

Results (0.22s):
       1 error
```

--- a/setup.py
+++ b/setup.py
@@ -22,5 +22,7 @@ setup(
     packages=["openapi3"],
     license="BSD 3-Clause License",
     install_requires=["PyYaml", "requests"],
-    tests_require=["pytest", "pytest-asyncio", "uvloop", "hypercorn", "pydantic", "fastapi"],
+    extras_require={
+        "test": ["pytest", "pytest-asyncio==0.16", "uvloop", "hypercorn", "pydantic", "fastapi"],
+    },
 )

